### ISSUE TITLE
Morphling facet "FLOW" cooldown reduction doesn't work

### ISSUE THREAD

**Issue by SSSsssplode** [ValveSoftware/Dota2-Gameplay#19204]
### Ability name

Attribute shift

### Description

tonight (2 CEST), after the last update morphling's facet "FLOW" is not working properly. it shows that your spells cooldown faster, but in fact it does not. 
![dota2_IE9dbFpf4o](https://github.com/ValveSoftware/Dota2-Gameplay/assets/96047157/1b904bfd-40fb-4d5f-984f-7a7782c20bca)
![dota2_XAZbo6aFDR](https://github.com/ValveSoftware/Dota2-Gameplay/assets/96047157/1d24da4b-56df-4d87-a923-d44bccec80c9)

### Example Match ID (and possibly Timestamp)

7781603640 - 6:07

**Comment by buttersome** [comment]
I noticed that too. I tried on Demo, having an observer ward to mark a starting point and use another one to mark a point where it takes morphing 12 seconds(duration of cd before talent) to walk. Shift queue the walk and wave into the ward and walk from the starting ward to the end point ward. Do this with both max strength and max age. If one reduces the cd faster, surely your cd would be refreshed in the half way point between the 2 wards. And the result is:
If you shift max agility, the cd does show that it is at 6secs, approximately 50-60% since they round it up to seconds, but you'd notice that in both cases, the cd only actually refreshes by the time you reach the end point, which means even though the icon shows the cd is reduced, it takes the same amount of time for it to actually be refreshed. 

I've also tried just spawning another morphing  ally, have one in str and other in age, and cast wave at the same time, and the result is the same, the cd shows that its reduced, but in reality it refreshes at the same time

Also if I'm not msitaken, before the most recent patch, the tooltip description was not very clear, the wording didn't quite make sense regarding the ratio, and I've played morphing during the 7.36 and 7.36a, and if I remember correctly, shifting to max strength gives you the accelerated CD.

Since 7,.36b, the description is now clear, that AT 175% agility, it is at its max accelerated CD. But I tried playing it, and realise it just feels different, until I noticed the CD doesn't reduce on 175% strength, and is now on 175% agility, but it still felt off. So I tested it out in demo.
